feature/acl/arp-policer/README.md

```diff
@@ -0,0 +1,57 @@
+
+# Interface based ARP policer
+
+## Summary
+
+This test ensures that NOS is able to police the ARP traffic on an interface.
+
+## Procedure
+Test environment setup
+
+### Topology
+Create the following connections:
+```mermaid
```

Comment:
I am unsure if we can view this as a topology or as a formula in codesearch.

Reply:
It renders in github markdown, so I think we can accept it.

Reply:
<img width="1116" alt="image" src="https://github.com/user-attachments/assets/cac13a5a-7505-4e34-beea-fbca6ee60927">